schema-registry/src/main/scala/net.manub.embeddedkafka/schemaregistry/package.scala

```diff
@@ -0,0 +1,64 @@
+package net.manub.embeddedkafka
```

Comment:
I'd move this to `net.manub.embeddedkafka.schemaregistry` to be consistent with the one in `embeddedKafka` module.